README.md

```diff
@@ -45,31 +45,7 @@ OpenDevin agents collaborate with human developers to write code, fix bugs, and
 ![App screenshot](./docs/static/img/screenshot.png)
 
 ## ⚡ Getting Started
-The easiest way to run OpenDevin is inside a Docker container. It works best with the most recent version of Docker, `26.0.0`.
-You must be using Linux, Mac OS, or WSL on Windows.
-
-To start OpenDevin in a docker container, run the following commands in your terminal:
-
-> [!WARNING]
-> When you run the following command, files in `./workspace` may be modified or deleted.
-
-```bash
-WORKSPACE_BASE=$(pwd)/workspace
-docker run -it \
-    --pull=always \
-    -e SANDBOX_USER_ID=$(id -u) \
-    -e WORKSPACE_MOUNT_PATH=$WORKSPACE_BASE \
-    -v $WORKSPACE_BASE:/opt/workspace_base \
-    -v /var/run/docker.sock:/var/run/docker.sock \
-    -p 3000:3000 \
-    --add-host host.docker.internal:host-gateway \
-    --name opendevin-app-$(date +%Y%m%d%H%M%S) \
-    ghcr.io/opendevin/opendevin:0.7
```

Comment:
How about we still keep this command but change the version to `latest` here? Personally. I still feel having a command to copy and paste on the `README.md` is kinda convenient.

Reply:
You want the entire section to remain in the README? Do you want me to update it to be exactly the same as the documentation?

Reply:
We just had the docs updated earlier. I'd prefer the main docker command to stay as well.

Reply:
The docs right now are not exactly the same as the README.
This PR: https://github.com/OpenDevin/OpenDevin/pull/2804 only changed it in the docs but not in the README. 

Should we keep everything the same as right now in the README and just remove the 0.7 tag? Or should we align it to be exactly as the docs?

Reply:
I wouldn't mind the README to be the same as the content in 2804. the extra explanations should avoid some confusion for new users.

Reply:
Agreed. 
The problem I'm trying to solve is the duplication. Just like the PR, one of the places keeps getting missed. I'll just keep it as is right now until we can come. Just removed the tag

Reply:
Agree with you, too. The de-duplication might just be a feature we aren't using yet with the docs?
Good task for a "good first issue" PR maybe.